### PR TITLE
More doc and fixes

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/docs/03_account.md [hunk 1]
```diff
@@ -9,209 +9,233 @@ It is responsible for the creation and management of account addresses and their
...
 ### `mint-nft`
 
 #### Parameters
 
-| Name    | Optional  | Default       | Example |
-| ------- | --------- | ------------- | ------- |
-| `` | ✓ | | |
+| Name                  | Optional  | Default                           | Example                                                           |
+| --------------------- | --------- | --------------------------------- | ----------------------------------------------------------------- |
+| `address`             | ✓         | The latest address of the account | "rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3" |
```
Comment (Thoralf-M) [inline documentation/docs/03_account.md]: Default is the first address

File: documentation/docs/03_account.md [hunk 2]
```diff
@@ -9,209 +9,233 @@ It is responsible for the creation and management of account addresses and their
...
+#### Example
 
 ```sh
-> Account "main": send
+> Account "main": send rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3 1000000
 ```
 
 ### `send-micro`
 
+Sends a micro amount to an address.
```
Comment (Thoralf-M) [inline documentation/docs/03_account.md]: Maybe add `with StorageDepositReturn and Expiration Unlock Conditions`?

File: documentation/docs/03_account.md [hunk 3]
```diff
@@ -9,209 +9,233 @@ It is responsible for the creation and management of account addresses and their
...
+#### Example
 
 ```sh
-> Account "main": send-micro
+> Account "main": send-micro rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3 1
 ```
 
 ### `send-native-token`
 
+Sends native tokens to an address.
```
Comment (Thoralf-M) [inline documentation/docs/03_account.md]: Here also maybe with `with StorageDepositReturn and Expiration Unlock Conditions`?

File: documentation/docs/03_account.md [hunk 4]
```diff
@@ -9,209 +9,233 @@ It is responsible for the creation and management of account addresses and their
...
 ```
 
 ### `sync`
 
-#### Parameters
-
-| Name    | Optional  | Default       | Example |
-| ------- | --------- | ------------- | ------- |
-| `` | ✓ | | |
+Synchronises all accounts.
```
Comment (Thoralf-M) [inline documentation/docs/03_account.md]: That's happening if called from the account manger, from the account it will only sync the selected one

Comment (thibault-martinez) [reply documentation/docs/03_account.md]: Failed copy paste, thanks

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**